tests-e2e/util/VideoElement.ts

```diff
@@ -5,7 +5,7 @@ export class VideoElement {
   readonly container: Locator
 
   constructor(page: Page) {
-    this.container = page.locator(".video-page")
+    this.container = page.locator(".video-embed, .video-page")
   }
 
   tab(opts: ByRoleOptions): Locator {
```

Comment:
The reason I suggested adding an `nth` method on VideoElement is that I thought it might be confusing to call `nth()` on the "final" locators (I.e., `tab().nth()`, `downloadButton().nth()`, ...).

E.g., imagine multiple video elements on a page, each with multiple tabs. Then `videoElement.tab()` would return something like 
```
video1_tab1, video1_tab2, video1_tab3, video2_tab1, video2_tab2, ...
```
So scoping the `nth` to the container seemed like a good idea. I.e., "Select tabs for the nth video" not "select the nth tab for all the videos".

**In practice this may not be a problem** since we only get multiple videos with embedded videos, and they only have a single tab anyway.